app/src/main/res/layout/exception_item.xml

```diff
@@ -48,8 +48,8 @@
 
     <ImageButton
         android:id="@+id/delete_exception"
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
+        android:layout_width="48dp"
```

Comment:
Was this meant to be `component_exceptions_learn_more_height` as well?

Reply:
I think this is specific to the exception item icon, we could add a `dimen` for it

Reply:
I added `component_tp_exceptions_icon_size` for it.

Reply:
Ah sorry, I misread. The dimen was super necessary, but it's useful. :)

Reply:
👍🏽